### DISCUSSION TITLE
Import from one installation to another

### DISCUSSION THREAD

**Discussion by danraymond** [kimai/kimai#3074] [Questions & Answers]
Hello,
I am a self-hosted user. I started on a local server on my PC. I have now  installed on my VPS so I can access from anywhere.

I want to import all the data from the first installation to the new one. I can see that I can use something like: "bin/console kimai:import:customer --help" However I get a "Permission denied" error (I am user root on my terminal)

How can I:
1. export ALL the data from the PC installation 
2. overcome the permission denied error on the VPS

Thank you

**Comment by empx92** [comment]
Did you manage it meanwhile?

**Comment by kevinpapst** [comment]
Everything is in here: https://www.kimai.org/documentation/backups.html

You have to copy the database over to the new server and point your new installation to it via the `.env` file

**Reply by empx92** [reply]
thanks for your reply, would it also work if 1st database is mysql and new one is mariadb ;) ?

**Reply by kevinpapst** [reply]
Yes, very likely. Remember to follow the update and not the installation process on the new server.

**Comment by danraymond** [comment]
Yes, I followed the instructions. Just be sure you have the correct
permissions on the VPS database, i.e. same user.
